### PR TITLE
Update module

### PR DESCRIPTION
**Pull request by m1ga**
Updating the module

* manifest
* library
* gradle file
* setter for `options` so you can asign the options later on too

[ti.bottomsheet-android-2.0.0.zip](https://github.com/hansemannn/titanium-android-bottom-sheet-dialog/files/11141977/ti.bottomsheet-android-2.0.0.zip)

tested by @jordanbisato

### CODE REVIEW

**Comment by hansemannn** [comment]
Any notable changes?

**Comment by m1ga** [comment]
Library:
    Migrated library to Android X.
    Updated dependency "AndroidUtil" to version 2.0.0.
    Updated targetSdkVersion to 28.

and you can use it with current emulators again since it is including the x86_64 version now. You couldn't compile to a device/emulator.

But I only compiled it for Jordan :smile:

**Comment by m1ga** [comment]
added `optionDlg.options = [...]` so you can fill it later and not only when creating it

File: android/build.gradle [hunk 1]
```diff
@@ -1,17 +1,7 @@
...
 
 dependencies {
-    compile 'com.github.michael-rapp:android-bottom-sheet:1.4.6'
-}
-
-task getDeps(type: Copy) {
-	from sourceSets.main.runtimeClasspath
-	exclude 'support-*'
-	into 'lib/'
+    implementation 'com.github.michael-rapp:android-bottom-sheet:2.0.0'
```
Comment (hansemannn) [inline android/build.gradle:6]: What are the changes in the depending library?

**Review by hansemannn — APPROVED** [review]

**Comment by hansemannn** [comment]
@m1ga Thank you! Is the attached binary up to date?

**Comment by m1ga** [comment]
yes, but just in case. This is the latest version:
[ti.bottomsheet-android-2.0.0.zip](https://github.com/hansemannn/titanium-android-bottom-sheet-dialog/files/11145308/ti.bottomsheet-android-2.0.0.zip)

**Pull request merged**